types/animals/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for animals 0.0
+// Project: https://github.com/boennemann/animals
+// Definitions by: Ben Cooper <https://github.com/bencooper222>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function main(): string;
+declare namespace main {
+    const words: string[];
+}
+export = main;
```

Comment:
this works, but looks like direct portamento from Java,
try this:
```suggestion
declare const animals: {
    (): string;
    readonly words: string[];
}

export = animals;
```

Reply:
Done